test_runner/fixtures/zenith_fixtures.py

```diff
@@ -1031,12 +1031,18 @@ def check_restored_datadir_content(zenith_cli: ZenithCli, test_output_dir: str,
     pg.stop()
 
     # Take a basebackup from pageserver
-    restored_dir_path = os.path.join(test_output_dir, "{}_restored_datadir".format(pg.branch))
+    restored_dir_path = os.path.join(test_output_dir, f"{pg.branch}_restored_datadir")
     mkdir_if_needed(restored_dir_path)
 
-    cmd = f"psql -h localhost -p {pageserver_pg_port} -c 'basebackup {pg.tenant_id} {timeline}' | tar -x -C {restored_dir_path}"
+    psql_path = os.path.join(pg.pg_bin.pg_bin_path, 'psql')
 
-    cmd = os.path.join(pg.pg_bin.pg_bin_path, cmd)
+    cmd = rf"""
+        {psql_path}                                    \
+            --no-psqlrc                                \
+            postgres://localhost:{pageserver_pg_port}  \
+            -c 'basebackup {pg.tenant_id} {timeline}'  \
+         | tar -x -C {restored_dir_path}
+    """
```

Comment:
I see that we're using `List[str]` for cmd arguments in most other places. We could possibly use two `Popen` with `PIPE` to allow that, but I'm not sure if it's worth it.

Reply:
Sure, that's a proper but also a cumbersome way to do this. I won't mind if someone applies the change you propose.